docs/development.md

```diff
@@ -63,6 +63,25 @@ bundle exec foreman start
 
 When using the dev server on Firefox you need to accept the self-signed certificate. You should go to `https://centos7-devel.example.com:3808` (or your equivalent) and add an exception for this certificate. Chrome does not throw any warnings if you have accepted the same certificate already for `https://centos7-devel.example.com`.
 
+### Reviewing Pull Requests
+
+It's easy to checkout pull requests from projects that were installed in development environment. All projects are cloned in vagrant's home, e.g. ~/foreman, ~/katello etc. In order to apply some PR to any of this project, you can use a reviewer script. See following example
+
+```sh
+cd ~/katello
+rpr 5266
+```
+
+`rpr` is shortcut for review pull request. This fetches information about PR number 5266, defines new remote in you .git/config, fetches new objects from it. Then it creates new local branch called review/pr5266 and pulls the code from the PR. It uses SSH connection so using ssh-agent, with SSH key uploaded to your github account, is recommended. If you're applying a PR on remote machine to which you connected through SSH, make sure you open that connection with agent forwarding, e.g. `ssh -a vagrant@host.example.com`. Don't forget to migration and seed the database if the PR contains related changes. Note that this might create new merge commit so git might want you to set your email and name. You can set that with following commands.
```

Comment:
We have a setting for that in forklift, but it'd be nicer if we didn't need SSH access to pull in changes. Especially since it's available over plain HTTP.

Reply:
this is reviewer feature and we can discuss it there. The problem with plain HTTP is that single IP can do limited number of queries during the day. From my company network, it's usually depleted early in the morning. That's why we use authenticated API. I wanted to avoid people having to provide credentials or tokens but there's already RFE https://github.com/ares/reviewer/issues/3 for that. So perhaps we could add more options later.